packages/canvas/canvas-renderer/src/CanvasRenderer.ts

```diff
@@ -38,16 +68,16 @@ export class CanvasRenderer extends AbstractRenderer
      * @param {number} [options.backgroundColor=0x000000] - The background color of the rendered area
      *  (shown if not transparent).
      */
-    constructor(options, arg2, arg3)
+    constructor(options?: IRendererOptions)
     {
-        super(RENDERER_TYPE.CANVAS, options, arg2, arg3);
+        super(RENDERER_TYPE.CANVAS, options);
 
         /**
          * The root canvas 2d context that everything is drawn with.
          *
          * @member {CanvasRenderingContext2D}
          */
-        this.rootContext = this.view.getContext('2d', { alpha: this.transparent });
+        this.rootContext = this.view.getContext('2d', { alpha: this.transparent }) as any;
```

Comment:
Why `as any`?

Reply:
Because line is too long

Reply:
So break it into two. Using `any` should be a last-resort for resolving typing issues, not workaround linting